pkg/loki/metrics_query.go

```diff
@@ -85,6 +93,8 @@ func NewTopologyQuery(cfg *Config, start, end, limit, metricFunction, metricType
 func getFields(scope, groups string) string {
 	var fields []string
 	switch scope {
+	case "app":
```

Comment:
I misunderstood what this "app query" was when you first explained to me. In my mind, "app"  rather refers to any applications running on cluster (like the `app` of `app.kubernetes.io/name` in k8s labels), so I was a little confused, but here it will sum all the flows for the given filters, right? (because there's only a single "app" label that we use in loki, at least at the moment.). Maybe it should be renamed "all" instead?